File: src/main/resources/static/js/service/processoService.js
angular.module("lexfirma").factory("processoAPI", function ($http) {
    const _getProcessos = function (idUsuario) {
        return $http.get(`/processos/${idUsuario}`)
    }
    const _saveProcesso = function (processo) {
        return $http.post(`/processo`, processo)

    }
    const _deleteProcesso = function (id) {
        return $http.delete(`/processo/${id}`)
    }
    return {
        getProcessos: _getProcessos,
        saveProcesso: _saveProcesso,
        deleteProcesso: _deleteProcesso
    }
})

File: src/main/java/com/fook/lexfirma/service/ProcessoService.java
package com.fook.lexfirma.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

import com.fook.lexfirma.model.Processo;
import com.fook.lexfirma.repository.ProcessoRepository;

@Component
public class ProcessoService {

    @Autowired
    private ProcessoRepository processoRepository;

    public Processo save(Processo processo) {
	return processoRepository.save(processo);
    }

    public List<Processo> getProcessos(Long usuarioID) {
	return processoRepository.findAllByUsuarioID(usuarioID);
    }

    public void deleteProcesso(Long id) {
	processoRepository.deleteById(id);
    }
}
